### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Rayzeq**
This pull request fixes bugs and compatibility issues with other plugins. If you have any questions about the reason for some changes, feel free to ask. The code has also been auto-formatted, if you want me to undo these changes, please ask me and I'll do it.
Notice: compatibility with gnome 43 has not been checked and is probably broken.

### CODE REVIEW

**Comment by qwreey75** [comment]
I will test it on 43! and we can create version check code if it not working

File: libs/dndQuickToggleHandler.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const DndQuickToggle = GObject.registerClass(
   class DndQuickToggle extends QuickToggle {
     _init() {
       super._init({
-        label: _('Do Not Disturb'),
+        title: _('Do Not Disturb'),
```
Comment (marcinjahn) [inline libs/dndQuickToggleHandler.js:9]: I was about to create a PR with that fix, thanks for saving my time :)

**Comment by marcinjahn** [comment]
Maybe it would make sense to split this PR into a few smaller ones? It seems it will take some time until such a big chunk of changes gets merged in.

**Comment by Rayzeq** [comment]
Technically, there is no big change, it's just a bunch of small changes.
As far as compatibility with g43 is concerned, there will be issues at some point anyway (for example, I'm not sure that the change from `label` to `title` is compatible with g43, but it's necessary for g44).

**Pull request merged**